src/cloud/project/project-patch.md

```diff
@@ -221,8 +221,8 @@ php ./vendor/bin/ece-patches revert
 This command reverts all patches in the following order:
 
 -  Reverts all applied custom patches from the /m2-hotfixes directory.
--  Reverts all applied Magento Quality patches.
--  Reverts all applied Magento Cloud patches.
+-  Reverts all applied optional individual patches.
+-  Reverts all applied required patches.
```

Comment:
Wanted to double-check this-- when I spoke to @NadiyaS I thought she mentioned that you cannot revert the required Cloud patches using the revert command.

Reply:
@meker12 This command was added as part of MQP story https://github.com/magento/magento-cloud-patches/blob/develop/src/Command/Ece/Revert.php
And as I see this command revert only Cloud patches

Reply:
@viktym, does the command revert optional individual patches too?

Reply:
Just talked to @viktym  
This command reverts all existed patches including quality too. We just need to rephrase the description of it in ece-patches.
@viktym please confirm

Reply:
Ok great. Thanks everyone.

Reply:
Followed up with Nadiia to confirm the change. Closing this.